### ISSUE TITLE
it seems, that support for Adobe Digital Editions has been broken

### ISSUE THREAD

**Issue by gregjozk** [nvaccess/nvda#15357]
Some users in Slovenia are reporting, that they cannot use ADE with NVDA and Narrator. It can be used with JAWS only.
When I tried to reproduce error, I have already found problems within instalation process (see attached log). So I cannot test behaviour within application, but as users reported me, NVDA cannot be used also within installed application (ADE).

### Steps to reproduce:

- start NVDA
- start instalation of ADE (ver 4.5.12
- try to perform commands like tab etc

### Actual behavior:

NVDA chrashs, NVDA cannot move focus to the next/previous field, NVDA cannot check/uncheck checkboxes etc

### Expected behavior:

normal behaviour within instalation process, normal behaviour during use of it, so book can be downloaded, (encreapted decreapted), and read.

#### NVDA installed/portable/running from source: 

latest alpha portable (its log is attached) and 2022.4 installed - same problems with addons disabled and enabled on both versions

#### NVDA version:

latest portable alpha and 2022.4 installed

#### Windows version:

Windows 10 22H2 (AMD64) build 19045.3324

#### Name and version of other software in use when reproducing the issue:

Adobe Digital editions 4.5.12

#### Does the issue still occur after restarting your computer?

yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

yes (see description)

#### If NVDA add-ons are disabled, is your problem still occurring?

yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

yes

[DigitalEditionsNVDA.log](https://github.com/nvaccess/nvda/files/12504876/digitaleditionsnvda.log)

**Comment by seanbudd** [comment]
Have you reported this to Adobe yet?

**Comment by gregjozk** [comment]
Not yet... Don't know how, ... where is their issue tracker for ADE
and other their applications.

2023-09-05 1:50 GMT+02.00, Sean Budd ***@***.***>:

**Comment by seanbudd** [comment]
I found this page by googling "adobe bug report": https://www.adobe.com/products/wishform.html

**Comment by gregjozk** [comment]
Thanks for your googling, but on this page I cannot find ADE. There is
Audition, Photoshop, Acrobat, Premiere, but not Digital editions. So I
cannot submit a bug report.

regards
Jožef

2023-09-05 3:23 GMT+02.00, Sean Budd ***@***.***>:

**Comment by seanbudd** [comment]
I would encourage contacting Adobe to mention this.  There's a variety of methods to contact Adobe on their website.
I assume you can still submit an issue using that form even if you select a different product in the dropdown.

**Comment by seanbudd** [comment]
Is this still an issue with the latest version of NVDA and ADE? Has this been reported to Adobe yet?

**Comment by seanbudd** [comment]
Also please confirm that this can be reproduced with an installed version of NVDA 2024.1. This is expected behaviour for a portable copy

**Comment by XLTechie** [comment]
@seanbudd wrote:
> Also please confirm that this can be reproduced with an installed version of NVDA 2024.1. This is expected behaviour for a portable copy

I just checked the user guide, and found no such mention. Should we?

**Comment by seanbudd** [comment]
@XLTechie - the use case here describes the installer being inaccessible. The assumption here is that this is during UAC elevation, where the software is running with administrator privileges. Portable copies are known to not work in these scenarios

**Comment by XLTechie** [comment]
@seanbudd I had not read this carefully, my apology.

**Comment by seanbudd** [comment]
There's been no updates from the author, closing until required information has been provided
